web/client/components/TOC/DefaultLayer.jsx

```diff
@@ -22,6 +22,7 @@ import ToggleFilter from './fragments/ToggleFilter';
 import tooltip from '../misc/enhancers/tooltip';
 import localizedProps from '../misc/enhancers/localizedProps';
 import { isInsideResolutionsLimits } from '../../utils/LayersUtils';
+import WFSLegend from './fragments/WFSLegend';
```

Comment:
Looking at the code I think the legend is more related to the style rather than the layer style so I propose to change the name WFSLegend to `VectorStyleLegend` where vector refer to all features rendered client side